sarahatiqah/dfsms
# Improve Product List Browsing Experience with Flexible Search Options

# Changes Made

1. Allow users to view the complete list of products without requiring a specific product name or performing a search.
2. Made the product table responsive.

# Details
<img width="1440" alt="Screenshot 2023-11-24 at 12 41 54 AM" src="https://github.com/sarahatiqah/dfsms/assets/76651485/f5fafb18-3d9f-4880-a7bb-cc79bc5d2c73">


# Testing
1. Clicks the "Search Product" option on the navbar.
2. Ensure that the product table is displayed without the need for a specific search query.
3. Resize your browser window or test on different devices to confirm that the table remains responsive and adjusts to different screen sizes.
